### ISSUE TITLE
Arrow Function with Rest Parameter

### ISSUE THREAD

**Issue by lagden** [eslint/eslint#3425]
I have this script tested on Firefox Developer Edition, but the `eslint` accuses error:

```javascript
function exec(fn) {
  return (...n) => Reflect.apply(fn, undefined, n);
}

function plus(...v) {
  return v.reduce((a, b) => a + b);
}

const m = exec(plus);
m(1, 2, 3); // 6
```
:finnadie: **error Unexpected parentheses around single function argument  arrow-parens**

Ok, let's fix it...

```javascript
function exec(fn) {
  return ...n => Reflect.apply(fn, undefined, n);
}
```
:finnadie: **error  Unexpected token ..**

huuummm... and now?
Is missing some rule?
Which is the solution for that?

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by michaelficarra** [comment]
This is already fixed in master.

**Comment by lagden** [comment]
Nice! Tks...
I'll be waiting for the next release.
